shiny/ui/_loading_spinners.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+
+from htmltools import HTMLDependency
+
+from .. import __version__
+
+
+def use_loading_spinners(
+    type: Optional[str] = "tadpole",
+    color: Optional[str] = None,
+    size: Optional[str] = None,
+    speed: Optional[str] = None,
+) -> HTMLDependency:
+    """
+    Use spinners to indicate when an element is loading.
+
+    When supplied in your app's UI, elements that are loading (e.g. plots or tables)
+    will have a spinner displayed over them. This is useful for when you have a
+    long-running computation and want to indicate to the user that something is
+    happening beyond the default grayed-out element.
+
+    Args:
+        `type`: The type of spinner to use. Options include "disc", "tadpole", and "dots".
+          Defaults to "tadpole".
+        `color`: The color of the spinner. This can be any valid CSS color. Defaults to
+          the current app "primary" color (if using a theme) or light-blue if not.
+        `size`: The size of the spinner. This can be any valid CSS size. Defaults to
+          "80px".
+        `speed`: The amount of time for the spinner to complete a single revolution. This
+          can be any valid CSS time. Defaults to "1s".
+    """
```

Comment:
This should be formatted numpydoc-style. I believe that's necessary for quartodoc to work properly.

For example, see https://github.com/posit-dev/py-shiny/blob/1611d5f6e509b36b62e57bb7eaf1702bcd2effbe/shiny/ui/_input_text.py#L24-L64